src/ui/util.coffee

```diff
@@ -191,8 +192,13 @@ insertTextAtCursor = (el, text) ->
         range.text = text
         range.select()
 
+autoLauncher = new AutoLaunch({
+    name: 'YakYak',
+    path: '/Applications/YakYak.app'
```

Comment:
Ah yes! I forgot to remove that parameter. I've removed it in a subsequent commit.

I've also tested it on windows server 2012 and it seems to work fine now.